### PR TITLE
updated to 5.0-b2

### PR DESCRIPTION
**Pull request by zerothi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by zerothi** [comment]
@conda-forge-admin, please rerender

**Comment by zerothi** [comment]
@conda-forge-admin, please rerender

**Review by jan-janssen — APPROVED** [review]
looks good to me

File: recipe/build.sh [hunk 1]
```diff
@@ -109,4 +118,21 @@ echo ">>>>>>>"
...
+# or unset these.
+export OMPI_MCA_plm=isolated
+export OMPI_MCA_btl_vader_single_copy_mechanism=none
+export OMPI_MCA_rmaps_base_oversubscribe=yes
+
+
+echo "Running tests"
+pushd obj_cmake/Tests/08.GeometryOptimization
+SIESTA_TESTS_VERIFY=1 ctest -L simple || echo "Accepted fail!"
+popd
```
Comment (jan-janssen) [inline recipe/build.sh:138]: Is there a specific reason to move the test from `run_test.sh` to `build.sh`?

Comment (zerothi) [reply recipe/build.sh:138]: because conda cleans source dirs, and I need the cmake build dir... :(

**Pull request merged**